worker/src/lib/dbLogger/DBLoggable.ts

```diff
@@ -858,6 +885,20 @@ export async function logRequest(
         }
       : unsupportedImage(requestBody);
 
+    const requestAssets: Record<string, string> = {};
```

Comment:
We can leave for now, but in the future when we add response asset support we should extract the traversing logic to a util.

Reply:
@colegottdank make sense! Gonna change this in next pr